packages/databricks-vscode/src/extension.ts

```diff
@@ -106,30 +105,17 @@ export async function activate(
     // Configuration group
     const connectionManager = new ConnectionManager(cli);
 
-    const workspaceFsDataProvider = new WorkspaceFsDataProvider(
-        connectionManager
-    );
     const workspaceFsCommands = new WorkspaceFsCommands(
         workspace.workspaceFolders[0].uri,
-        connectionManager,
-        workspaceFsDataProvider
+        connectionManager
     );
 
     context.subscriptions.push(
-        window.registerTreeDataProvider(
-            "workspaceFsView",
-            workspaceFsDataProvider
-        ),
         commands.registerCommand(
             "databricks.wsfs.attachSyncDestination",
             workspaceFsCommands.attachSyncDestination,
             workspaceFsCommands
         ),
-        commands.registerCommand(
```

Comment:
We can remove wsfs.attachSyncDestination as well and remove these commands from package.json.

Reply:
+1 will do